docs/src/examples/components/Menu/Variations/MenuExampleTabList.shorthand.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Menu, TabBehavior, TabListBehavior } from '@stardust-ui/react'
+
+const items = [
+  { key: 'editorials', content: 'Editorials', accessibility: TabBehavior },
+  { key: 'review', content: 'Reviews', accessibility: TabBehavior },
+  { key: 'events', content: 'Upcoming Events', accessibility: TabBehavior },
+]
```

Comment:
Should the accessibility prop always be the same across all MenuItems inside one Menu? If this is the case, we can just apply the accessibility prop to the Menu and provide the same as defaultProperty to all MenuItems inside the Menu. Take a look into the renderItem method in the Menu and decide if we should do this in the similar way as the other common props are delegated to the MenuItem components.

Reply:
yes, agree - we've discussed this moment, and agreed that it will be addressed by means of separate PR. For sure, there shouldn't be a mandatory requirement for consumer to specify this thing for menu items (especially given that those menu item objects could come as result of deserialising some HTTP response data, and at that moment there could be no context provided for them about which behavior Menu is using).

The way it could be achieved is by introducing `behaviors` property to Accessibility Behavior: 

```js
const behavior = {
  attributes: { .. },
  behaviors: { item: TabBehavior }
}
```

We should create and address RFC (issue) for that.

Reply:
That a good point! Just one thing to keep in mind, that we should leave the possibility to set different behaviors for Items (for e.g. case when one MenuItem has a submenu, others - don't).
So, in Menu, we can set default behaviors for items, unless a user has already set it.

Reply:
yes - but this option is always opened to us, as it is still possible to explicitly specify behavior for each individual menu item via `accessibility` prop:

```jsx
<Menu items=[
   ...
   { key: '...', content: '...', accessibility: CustomBehavior }
]
```

Here we are touching just the fact that _some reasonable default behavior_ should be applied to `Menu`/`List` items.